src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/GenericComponent.tsx

```diff
@@ -99,7 +99,11 @@ export function GenericComponent(props: IGenericComponentProps) {
 
     const dataModelBinding = props.dataModelBindings[key];
     if (props.triggers && props.triggers.includes(Triggers.Validation)) {
-      dispatch(setCurrentSingleFieldValidation({ dataModelBinding }));
+      dispatch(setCurrentSingleFieldValidation({
+        dataModelBinding,
+        componentId: props.id,
+        layoutId: currentView,
+      }));
     }
```

Comment:
Seems like the unit-test is not updated to reflect this change

Reply:
Yep, fixed the unit tests now :)